merlin/models/tf/blocks/core/transformations.py

```diff
@@ -27,15 +27,16 @@
 from merlin.models.tf.utils.tf_utils import (
     df_to_tensor,
     get_candidate_probs,
+    list_col_to_ragged,
     transform_label_to_onehot,
 )
 from merlin.models.utils import schema_utils
 from merlin.schema import Schema, Tags
 
 
-@Block.registry.register("as-sparse")
+@Block.registry.register("as-ragged")
 @tf.keras.utils.register_keras_serializable(package="merlin.models")
-class AsSparseFeatures(TabularBlock):
+class AsRaggedFeatures(TabularBlock):
     """
     Convert inputs to sparse tensors.
```

Comment:
💡 Updating the docstring here would help clarify that this is only applying to list-like columns. similar to the AsDenseFeatures block.